community/hacktoberfest/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: single
+title:  "Hacktoberfest"
+sidebar:
+  nav: community
+---
+
+[Hacktoberfest](https://hacktoberfest.digitalocean.com/) is a month-long global celebration of open source software that happens every year in October. During the month of October, you can contribute to Spinnaker and earn swag and recognition from the [Continuous Delivery Foundation](https://cd.foundation), our parent organization.
+
+We invite everyone to participate and to contribute to the Spinnaker project, regardless of their background and Spinnaker experience. There are many ways to contribute to Spinnaker during Hacktoberfest and other contributor events. You can:
+- Work on core or plugin code
+- Write and improve documentation
+- Assist community members in [Spinnaker Slack](http://join.spinnaker.io)
+- Answer [StackOverflow questions](https://stackoverflow.com/questions/tagged/spinnaker)
+- Share your experiences with Spinnaker by creating a new blog post, video, or social media steam 
+
+For more information on eligible contributions, see the [What to Hack](/community/gardening/what-to-hack/) page. For help getting started as a Spinnaker contributor, visit the [Contributing](/community/contributing/) page.
+
+All of the following GitHub events count as contributions, as long as they're made to the `spinnaker` and `spinnaker-hackathon` organizations:
+* pull requests
+* issues
+* comments on PRs and issues
+
+Here are some ideas to get you started:
+- [Beginner-friendly issues on GitHub](https://github.com/spinnaker/spinnaker/issues?q=is%3Aopen+is%3Aissue+label%3A%22beginner+friendly%22)
+- [Project ideas](https://github.com/spinnaker-hackathon/gardening/blob/master/project-ideas.md) from Gardening Days, the Spinnaker hackathon program
+
+### Hacktoberfest versus Gardening Days
+
+The Spinnaker community is experimenting with new ways to invite contributors into our community, and that may cause some confusion. Since launching the Spinnaker Gardening Days hackathon program, we have created new resources meant to assist contributors in those events and beyond. Thus, if you are participating in CDF's Hacktoberfest and explore Spinnaker contribution, you may see links to Gardening Days resources. Feel free to use those resources to help you get set up to contribute as part of Hacktoberfest. YOu can also use the [#gardening-general](https://spinnakerteam.slack.com/archives/CV4A90DPF) channel in [Slack](http://join.spinnaker.io) to ask questions about contributing and get to know the community.
```

Comment:
```suggestion
The Spinnaker community is experimenting with new ways to invite contributors into our community, and that may cause some confusion. Since launching the Spinnaker Gardening Days hackathon program, we have created new resources meant to assist contributors in those events and beyond. Thus, if you are participating in Hacktoberfest and explore Spinnaker contribution, you may see links to Gardening Days resources. Feel free to use those resources to help you get set up to contribute as part of Hacktoberfest. You can also use the [#gardening-general](https://spinnakerteam.slack.com/archives/CV4A90DPF) channel in [Slack](http://join.spinnaker.io) to ask questions about contributing and get to know the community.